docs/index.rst

```diff
@@ -0,0 +1,41 @@
+gnomad_qc
+=========
+This is the documentation for the complete set of scripts used to perform sample and
+variant QC for the gnomAD v4 release. We are continuously updating and improving upon
+the code to handle new releases as they grow in size and complexity and as they require
+increasingly sophisticated QC treatment. The current code therefore represents the most
+recent iteration of our pipelines and is guaranteed to change over time.
+
+The scripts make reference to gnomAD-related metadata files (not public) and may
```

Comment:
```suggestion
The scripts reference gnomAD-related metadata files (not public) and may
```
optional (slightly clearer)

Reply:
also maybe along these lines: should we mention that our QC is incredibly tailored to Broad-sequenced/processed data?

Reply:
yeah we can, to make this fast, it was really mostly a copy and paste of the repo readme